### PR TITLE
Elk 2 write event tests

### PR DESCRIPTION
**Pull request by elfenpiff**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## Required information
+
+**Operating system:**
+Output of: `uname -a`
+
+**Rust version:**
+Output of: `rustc --version`
+
+**elkodon version:**
+E.g. `v1.2.3` or `master` branch
```
Comment (elBoberido) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: @elfenpiff shouldn't this be master?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+---
+
+## Required information
+
+**Operating system:**
+Output of: `uname -a`
+
+**Rust version:**
+Output of: `rustc --version`
```
Comment (elBoberido) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: Would it make sense to to also have a cargo version?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+## Required information
+
+**Operating system:**
+Output of: `uname -a`
```
Comment (elBoberido) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: Maybe add that this is for Linux ... maybe also other Unix. Windows might have something similar

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+## Notes for Reviewer
+<!-- Items in addition to the checklist below that the reviewer should look for -->
+
+## Pre-Review Checklist for the PR Author
+
+1. [ ] Add sensible notes for the reviewer
+1. [ ] PR title is short, expressive and meaningful
+1. [ ] Relevant issues are linked
+1. [ ] Every source code file has the SPDX header `SPDX-License-Identifier: $(LICENSE_CODE)`
+1. [ ] Branch follows the naming format (`elk-123-introduce-posix-ipc-example`)
```
Comment (elBoberido) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: This is for budrus :rofl:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+1. [ ] Commits messages are according to this [guideline][commit-guidelines]
+    - [ ] Commit messages have the issue ID (`[#123] add posix ipc example`)
```
Comment (elBoberido) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: The example violates the guideline ;)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] Public API is documented
+- [ ] PR title describes the changes
+
+## Post-review Checklist for the PR Author
+
+1. [ ] All open points are addressed and tracked via issues
+
+## References
+
+- Closes **TBD**
```
Comment (elBoberido) [inline .github/PULL_REQUEST_TEMPLATE.md:37]: In iceoryx people quite often forget about this or have not idea what is meant. Could you please add a comment like `<!-- Use 'Closes #123' or 'Relates to #123' -->`

File: README.md [hunk 7]
```diff
@@ -1,39 +1,37 @@
...
- * https://jamey.thesharps.us/2017/01/03/which-projects-should-convert-to-rust/
- * https://serokell.io/blog/rust-companies
- * https://www.zdnet.com/article/programming-languages-its-time-to-stop-using-c-and-c-for-new-projects-says-microsoft-azure-cto/
-   * https://www.theregister.com/2022/09/28/is_it_time_to_retire_c/
- * https://medium.com/volvo-cars-engineering/why-volvo-thinks-you-should-have-rust-in-your-car-4320bd639e09
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/elboberido"><img src="https://avatars.githubusercontent.com/u/56729607" width="100px;" alt="Mathias Kraus"/><br /><sub><b>Mathias Kraus</b></sub></a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/elfenpiff"><img src="https://avatars.githubusercontent.com/u/56729169" width="100px;" alt="Christian Eltzschig"/><br /><sub><b>Christian Eltzschig</b></sub></a></td>
```
Comment (elBoberido) [inline README.md:31]: What do you think about adding the github name, e.g. `Mathias Kraus aka elBoberido` or `Mathias »elBoberido« Kraus` or `elBoberido - Mathias Kraus`?

File: doc/release-notes/elkodon-unreleased.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+### API Breaking Changes
+
+1. Example
+
+    ```rust
+    // old
+    let fuu = Hello().IsItMeYouAreLookingFor()
+
+    // new
+    let fuu = Hello().MyNameIsBlobb()
```
Comment (elBoberido) [inline doc/release-notes/elkodon-unreleased.md:36]: snake_case snake_case snake_case snake_case snake_case

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (elfenpiff) [reply .github/ISSUE_TEMPLATE/bug_report.md:19]: @elBoberido you mean `main`...

File: README.md (hunk 7 above)
Comment (elfenpiff) [reply README.md:31]: I will use the second version, then I have the feeling that we sound like mafia gangsters ;)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (elBoberido) [reply .github/ISSUE_TEMPLATE/bug_report.md:19]: Hahaha, yes

**Pull request merged**